server/util.go

```diff
@@ -43,7 +43,7 @@ const (
 
 // Version information.
 var (
-	PDReleaseVersion = "0.9.0"
+	PDReleaseVersion = "1.0.0+git"
```

Comment:
why +git here?

Reply:
Make it different from release branches, using the same method as etcd. https://github.com/coreos/etcd/blob/b989e1992fb1186c7a6a5ea9291fae888a3d7971/version/version.go#L29